src/Microsoft.SymbolStore/Microsoft.SymbolStore.csproj

```diff
@@ -15,10 +15,6 @@
 
   <ItemGroup>
     <PackageReference Include="System.Reflection.Metadata" Version="$(SystemReflectionMetadataVersion)" />
-<!--
```

Comment:
Remove this - and I'd say unless we have a reason to ship a min version (i.e. it's higher than .NET 6's) we should only ship this in FX?

Reply:
I did remove the already commented out section here.  What are you asking to do how?

Reply:
It looked green when I was reviewing :) I was probably tired - sorry.